core/server/data/importer/index.js

```diff
@@ -139,7 +140,16 @@ _.extend(ImportManager.prototype, {
             ),
             dirMatches = glob.sync(
                 this.getDirectoryGlob(this.getDirectories(), ROOT_OR_SINGLE_DIR), {cwd: directory}
-            );
+            ),
+            oldRoonMatches = glob.sync(this.getDirectoryGlob(['drafts', 'published', 'deleted'], ROOT_OR_SINGLE_DIR),
+                {cwd: directory});
+
+        // This is a temporary extra message for the old format roon export which doesn't work with Ghost
+        if (oldRoonMatches.length > 0) {
+            return Promise.reject(new errors.UnsupportedMediaTypeError(
```

Comment:
I think this is just a holdover from before `isValidZip` was changed to not return promises.